apps/src/javalab/PreviewPaneHeader.jsx

```diff
@@ -8,6 +8,7 @@ import PaneHeader, {
 } from '@cdo/apps/templates/PaneHeader';
 import CollapserIcon from '@cdo/apps/templates/CollapserIcon';
 import {RecordingFileType} from '../code-studio/components/recorders';
+import {white} from '../util/color';
```

Comment:
Nit: I had no idea you could import individual colors this way! I do see one other example of us importing colors like this (ie, grabbing a specific color), but the more common approach would just be to import the color object like this, then referencing the specific color in your style object:

https://github.com/code-dot-org/code-dot-org/blob/78190b48869febecc5d575787ab0eb05bbce78f0/apps/src/javalab/JavalabConsole.jsx#L5